pkg/ui/styl/layout/layout.styl

```diff
@@ -146,6 +146,7 @@ $page-padding-top   = 20px
     color $tooltip-color
     line-height 11px
     text-align center
+    cursor default
```

Comment:
Does it turn into a pointer otherwise?  That's too bad.  Perhaps something else to think about when refactoring.

Reply:
Turns into a text cursor, because of the "i". Not text or clickable, so should be the arrow IMO.

Reply:
Oh that's funny.